<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>handOverShiftWgt</class>
 <widget class="QWidget" name="handOverShiftWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>handOverShiftWgt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget{
background:#F8F8F8;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="pagePointListWgt" name="page_pointList"/>
        <widget class="pageSettlementWgt" name="page_settlement"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>pagePointListWgt</class>
   <extends>QWidget</extends>
   <header>pagePointListWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pageSettlementWgt</class>
   <extends>QWidget</extends>
   <header>pageSettlementWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
